src/test/setup.ts

```diff
@@ -1,9 +1,9 @@
-import nodeFetch, { Request, Response } from 'node-fetch';
+// import nodeFetch, { Request, Response } from 'node-fetch';
 
 import { server } from './mocks/server';
 import 'vitest-canvas-mock';
 
-Object.assign(global, { fetch: nodeFetch, Request, Response });
+// Object.assign(global, { fetch: nodeFetch, Request, Response });
```

Comment:
I've removed this and it changed nothing, no idea why it's here, any idea @regexowl ?
But IMO it is irrelevat to the fix.

Reply:
I've added the polyfills during the vitest migration as there were errors outputted without them. But it doesn't seem like the case anymore with bumped msw, so I'd say we can get rid of them :+1: